setup.py

```diff
@@ -58,6 +58,7 @@
     'arrow>=0.7.0',
     'Flask-BabelEx>=0.9.2',
     'Jinja2>=2.8',
+    'future>=0.16.0',
```

Comment:
It's probably a stupid question - but that does this library do? I don't see it imported anywhere, so I assumed it's for the `from __future__` import, but I thought this library is built in and you don't have to include it in `setup.py`?

Reply:
Ok, that was the part that I was missing: `future` is needed to do `from builtins import ...` [(source)](http://python-future.org/compatible_idioms.html#setup) 😄

Reply:
yep! anyways I am removing it, we will hash the badge title and value